.github/workflows/test-amazon-web-services-install.yaml

```diff
@@ -100,13 +100,26 @@ jobs:
     - name: Workaround - wait for all the k8s resources to be ready
       timeout-minutes: 15
       run: |
-        echo "Waiting for pods to be ready..."
-        while ! kubectl wait --for=condition=Ready pods --timeout=60s --all -n posthog > /dev/null 2>&1
+        echo "Waiting for jobs to be ready..."
```

Comment:
non-blocking

Worth pushing into a script, so it's easier to test locally, and also shared between aws and do?